.github/workflows/test-opentofu.yml

```diff
@@ -0,0 +1,67 @@
+# The primary focus of this provider is Terraform, however, it is nice to see if there are deviations in the implementations between Terraform and OpenTofu.
+# This is a restricted test set for OpenTofu
+# The full test is defined in the test.yml workflow. 
+# This file is intended to focus testing the OpenTofu compatibility against the latest provider version. 
+# This is a manually triggered test.
+
+name: OpenTofu Provider Tests
+
+on:
+  workflow_dispatch:
+
+# Testing only needs permission to read the repository contents.
+permissions:
+  contents: read
+
+  
+jobs:
+  # Ensure project builds before running the testing matrix
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    steps:
+      - uses: actions/checkout@v4 # v4.0.0
+      - uses: actions/setup-go@0a12ed9d6a96ab950c8f026ed9f722fe0da7ef32 # v5.0.2
+        with:
+          go-version-file: 'go.mod'
+          cache: true
+      - run: go mod download
+      - run: go build -v .
+      - name: Run linters
+        uses: golangci/golangci-lint-action@971e284b6050e8a5849b72094c50ab08da042db8 # v3.7.1
+        with:
+          version: latest
+
+
+  # Run acceptance tests in a matrix with OpenTofu CLI versions
+  test:
+    name: Terraform Provider Acceptance Tests for OpenTofu
+    needs: build
+    runs-on: ubuntu-latest
+    # Timeout for tests set to 25 minutes to safeguard long running tests (specifically for service instances) 
+    timeout-minutes: 25
+    strategy:
+      fail-fast: false
+      matrix:
+        # list of OpenTofu versions to test against
+        opentofu:
+          - '1.6.*'
+          - '1.7.*'
+          - '1.8.*'
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-go@0a12ed9d6a96ab950c8f026ed9f722fe0da7ef32
+        with:
+          go-version-file: 'go.mod'
+          cache: true
+      - uses: opentofu/setup-opentofu@v1
+        with:
+          tofu_version: ${{ matrix.opentofu }}
+          tofu_wrapper: false
+      - run: go mod download
+      - env:
+          TF_ACC: "1"
+        run: go test -v -cover -coverprofile=cover.out -timeout=1800s -parallel=4 ./...
```

Comment:
Shouldn't we give a timeout of 900s to replicate that of the terraform acceptance tests?

Reply:
agree